diff --git a/es/components/table/src/table-body/events-helper.mjs b/es/components/table/src/table-body/events-helper.mjs
index f6cc5f8dda6361b9cf459657efd5b516bc2614d6..e0fb31a24b158afb9cc37c93590f3c12f087a199 100644
--- a/es/components/table/src/table-body/events-helper.mjs
+++ b/es/components/table/src/table-body/events-helper.mjs
@@ -5,7 +5,7 @@ import { getCell, getColumnByCell, createTablePopper } from '../util.mjs';
 import { TABLE_INJECTION_KEY } from '../tokens.mjs';
 import { addClass, hasClass, removeClass } from '../../../../utils/dom/style.mjs';
 
-function isGreaterThan(a, b, epsilon = 0.01) {
+function isGreaterThan(a, b, epsilon = 0.03) {
   return a - b > epsilon;
 }
 function useEvents(props) {